BUILDING.md

```diff
@@ -99,8 +99,10 @@ $ node -e "console.log('Hello from Node.js ' + process.version)"
 Prerequisites:
 
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
-* Visual Studio 2013 / 2015, all editions including the Community edition, or
-* Visual Studio Express 2013 / 2015 for Desktop
+* Either one of:
```

Comment:
"Either" is normally applied to a choice of two. I'd drop this and go with `One of:`